golem-templates/templates/ts/ts-app-common/common-ts/golem.yaml

```diff
@@ -15,7 +15,8 @@ templates:
       - dist/main.js
     - generateQuickjsCrate: ../../golem-temp/quickjs-wrappers/{{ component_name | to_snake_case }}
       wit: wit-generated
-      js: dist/main.js
+      jsModules:
+        main: dist/main.js
```

Comment:
User can give any name to the entry point. If its `index.ts`, then I guess it will be `dist/index.js` . This is unrelated to agent specific template by the way

Reply:
Well this template calls it main. I don't understand the comment